src/main/java/com/tinkoff/edu/app/LoanCalcController.java

```diff
@@ -6,10 +6,11 @@
 
 public class LoanCalcController {
 
-    public static int createRequest() {
-        int requestId = LoanCalcService.createLoanRequest();
-        LoanCalcLogger.log(requestId);
-        return requestId;
+    public LoanResponse createRequest(LoanRequest request) {
+        LoanCalcService service = new LoanCalcService();
+        LoanResponse loanResponse = service.createLoanRequest(request);
+        LoanCalcLogger.log(loanResponse.getRequestId());
```

Comment:
можно немного дополнить логирование - 
```suggestion
        LoanCalcLogger.log("Created loan request with id " + loanResponse.getRequestId());
```

Reply:
так у меня внутри метода log более подробная строка вывода 

`public static void log(int requestId) {
        System.out.println("Logger working with loan number: " + requestId);
    }`